src/lib.rs

```diff
@@ -728,6 +741,16 @@ impl ConsoleEngine {
         None
     }
 
+    /// Give the terminal resize event
```

Comment:
```suggestion
    /// Give the terminal resize event
    ///
    /// usage:
    /// ```
    /// if let Some(width, height) = engine.get_resize() {
    ///     // do something
    /// }
```

Reply:
Yes, adding a example is good idea, but example snippet contains error

Here a right version
```suggestion
    /// Give the terminal resize event
    ///
    /// usage:
    /// ```
    /// if let Some((width, height)) = engine.get_resize() {
    ///     // do something
    /// }
    /// ```
```